cli/threshold.go

```diff
@@ -33,15 +35,20 @@ var createThresholdCmd = &cobra.Command{
 			logger.Fatal("failed to get keys count flag value", zap.Error(err))
 		}
 
+		if !ssvtypes.ValidCommitteeSize(int(keysCount)) {
+			logger.Fatal("invalid keys count", zap.Int("keysCount", int(keysCount)))
+		}
+
 		baseKey := &bls.SecretKey{}
 		if err := baseKey.SetHexString(privKey); err != nil {
 			logger.Fatal("failed to set hex private key", zap.Error(err))
 		}
 
 		// https://github.com/ethereum/eth2-ssv/issues/22
-		// currently support 4 nodes threshold is keysCount-1(3). need to align based open the issue to
+		// currently support 4, 7, 10, 13 nodes threshold 3f+1. need to align based open the issue to
 		// support k(2f+1) and n (3f+1) and allow to pass it as flag
-		privKeys, err := threshold.Create(baseKey.Serialize(), keysCount-1, keysCount)
+		f := ssvtypes.ComputeF(int(keysCount))
+		privKeys, err := threshold.Create(baseKey.Serialize(), f, keysCount)
```

Comment:
i think the threshold should be quorum not f

Reply:
@AKorpusenko I fixed it, please notice.